dll/win32/ucrtbase/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+
+spec2def(ucrtbase.dll ucrtbase.spec ADD_IMPORTLIB)
+
+add_library(ucrtbase SHARED
+    stubs.c
+    ucrtbase_stubs.c
+    ucrtbase.def
+    ${REACTOS_SOURCE_DIR}/sdk/lib/crt/startup/tlsmcrt.c # HACK
+    ${REACTOS_SOURCE_DIR}/sdk/lib/crt/startup/tlsmthread.c
+    ${REACTOS_SOURCE_DIR}/sdk/lib/crt/startup/tlsthrd.c
+)
+
+if(MSVC)
+    # Disable default libraries
+    target_link_options(ucrtbase PRIVATE "/NODEFAULTLIB:MSVCRTD")
+
+    # Set entrypoint
+    target_link_options(ucrtbase PRIVATE "/ENTRY:__acrt_DllMain")
```

Comment:
Why is this needed, since down below you unconditionally do `set_entrypoint(ucrtbase __acrt_DllMain 12)` ?

Reply:
It's not needed. it's a remainder from the initial code. Removed it.